src/Tickets/Commerce/Cart.php

```diff
@@ -339,8 +338,11 @@ public function get_items_in_cart( $full_item_params = false ) {
 					return null;
 				}
 
+				$sub_total_value = new Commerce\Utils\Value();
+				$sub_total_value->set_value( $item['obj']->price );
```

Comment:
We should create a static method that returns an instance of Value. A `make` of sorts.